library/public/types.h

```diff
@@ -58,6 +71,56 @@ class F3D_EXPORT ratio_t
   double Value;
 };
 
+/**
+ * Describe a RGB color.
+ */
+class F3D_EXPORT color_t
```

Comment:
@Meakk I only declared what we need for compiling F3D. Technically this is a break of retro compatibility of the libf3d API since color_t is not a std::vector. I think this is fine, let me know.

Reply:
I think it's fine if it's too complicated to keep retro compatibility. Let's document it properly in the release note.